README.md

```diff
@@ -7,20 +7,25 @@ Prepares a machine to be an Omnibus builder.
 
 This project is managed by the CHEF Release Engineering team. For more information on the Release Engineering team's contribution, triage, and release process, please consult the [CHEF Release Engineering OSS Management Guide](https://docs.google.com/a/opscode.com/document/d/1oJB0vZb_3bl7_ZU2YMDBkMFdL-EWplW1BJv_FXTUOzg/edit).
 
+Announcement
+------------
+Starting with omnibus cookbook version 4.0.0, Chef 11 is no longer supported, since 12.5.1 is required to use [chef-ingredient](https://github.com/chef-cookbooks/chef-ingredient). If needed, you can pin to omnibus version `3.2.5` to preserve the old functionality.
+
+Starting with 4.0.0, instead of compiling everything from source in this cookbook, the [omnibus-toolchain](https://github.com/chef/omnibus-toolchain) package will be installed. This package contains patch, bash, make, git, ruby, rubygems, and bundler (built from [omnibus-software](https://github.com/chef/omnibus-software) definitions).
+
```

Comment:
For readability, I would make this one announcement ("Starting with omnibus cookbook version 4.0.0"...) with two bullet points.